### DISCUSSION TITLE
A few wish list items

### DISCUSSION THREAD

**Discussion by tmumm** [templaza/astroid-framework#808] [Ideas]
I've built dozens of sites using Astroid, I love its flexibility and ease.   There are a few things that I keep wishing for: 

1.  It would be great if the color selectors used the "primary" and "secondary" "light" etc definitions.   Once set in the theme, those colors should be available on the palette.    Generally the theme settings should be able to be used more easily.   Then if changed in the theme, it can change universally.  

2.  MegaMenu - When selecting a top level menu item, the submenu items should show below it, not as a flyout menu.  I know I can add them one by one, but that is tedious and makes it so if the menu changes it's a multi-step process.  

3.  We often use the split menu, with logo on left and modules and menu on right.  I would love to see one more option with right alignment.  I can fix with CSS, but use it often.   This is a much more common use I believe. 
![Home](https://github.com/user-attachments/assets/006014e8-394f-441c-ad9b-cf4caf85451a)

Let me know if I can clarify and I appreciate your consideration.

**Comment by sonvnn** [comment]
Thanks for your ideas.
1. It's great. I will add this to tasklist
2. I will consider about it
3. Yes, I will think about another option for the next templates.
